group/var_ed25519/curve.go

```diff
@@ -67,7 +67,7 @@ func (c *curve) PointLen() int {
 	return (c.P.BitLen() + 7 + 1) / 8
 }
 
-// NewKey returns a formatted curve25519 key (avoiding subgroup attack by requiring
+// NewKey returns a formatted var_ed25519 key (avoiding subgroup attack by requiring
 // it to be a multiple of 8). NewKey implements the kyber/util/key.Generator interface.
 func (c *curve) NewKey(stream cipher.Stream) kyber.Scalar {
```

Comment:
This seems like an undesirable change. 
Also @ineiti @pierluca this function seems to be applying the clamping from Ed25519, which probably is not the right thing to do for all the other params provided in the param.go file... It might introduce bias or big issues.

Reply:
maybe `returns a scalar suitable for use as a secret key` instead of naming the curve @matteosz?